### PR TITLE
Create component Input

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Create component Input

image

### CODE REVIEW

**Comment by thanhnguyen-agilityio** [comment]
For input, pls take a look on how to design an input component here
https://v2.chakra-ui.com/docs/components/input#add-elements-inside-input

File: examples/practice-typescript/src/components/Input/index.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import './index.scss';
+
+interface InputProps {
+  type: string;
```
Comment (thanhnguyen-agilityio) [inline examples/practice-typescript/src/components/Input/index.tsx]: `type` is a keyword, pls avoid to use as name. Take a look https://github.com/microsoft/TypeScript/issues/2536

Comment (thanhtrantruong47) [reply examples/practice-typescript/src/components/Input/index.tsx]: Fixed

File: examples/practice-typescript/src/components/Input/index.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import '../../styles/component.scss';
+
+interface InputProps {
+  typeInput: string;
+  styleClass: 'search';
```
Comment (thanhnguyen-agilityio) [inline examples/practice-typescript/src/components/Input/index.tsx]: Pls recheck how to use an interface in TS. This is an object types but has the value.
https://www.typescriptlang.org/docs/handbook/2/objects.html

File: examples/practice-typescript/src/components/Input/index.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  outline: none;
+  max-width: 228px;
+  max-height: 40px;
+  margin-right: 85px;
+  padding: 11px 10px 10px 40px;
+  font-size: 16px;
+}
+
+.search-icon {
+  background-image: url(../assets/images/managers/icon-search.png);
```
Comment (thanhnguyen-agilityio) [inline examples/practice-typescript/src/components/Input/index.scss]: But why it is a background image? When a thing is a background and is it a background here?

**Review by thanhnguyen-agilityio — APPROVED** [review]

**Pull request merged**